yt_dlp/extractor/tver.py

```diff
@@ -53,6 +53,27 @@ def _real_extract(self, url):
                 (r'canonical"\s*href="(https?://tver\.jp/[^"]+)"', r'&link=(https?://tver\.jp/[^?&]+)[?&]'),
                 webpage, 'url regex'))
 
+        if video_type == 'olympic/paris2024/video':
+            olympic_api_info = self._download_json(
+                'https://tver.jp/olympic/paris2024/req/api/hook?q=https%3A%2F%2Folympic-assets.tver.jp%2Fweb-static%2Fjson%2Fconfig.json&d=',
+                video_id,
+                fatal=False,
+            )
+            p_id = traverse_obj(
+                olympic_api_info,
+                ('content', 'brightcove', 'E200', 'pro', 'pc', 'account_id'),
+                get_all=False,
+            )
+            r_id = video_id
+            return {
+                '_type': 'url_transparent',
+                'url': smuggle_url(
+                    self.BRIGHTCOVE_URL_TEMPLATE % (p_id, r_id),
+                    {'geo_countries': ['JP']},
+                ),
+                'ie_key': 'BrightcoveNew',
+            }
```

Comment:
- Since we are extracting the player ID from a static URL, we could instead just hardcode the ID and skip the extra request
- `url_transparent` is not needed here since we are not returning any additional metadata

```suggestion
            # Player ID is taken from .content.brightcove.E200.pro.pc.account_id:
            # https://tver.jp/olympic/paris2024/req/api/hook?q=https%3A%2F%2Folympic-assets.tver.jp%2Fweb-static%2Fjson%2Fconfig.json&d=
            return self.url_result(smuggle_url(
                self.BRIGHTCOVE_URL_TEMPLATE % ('4774017240001', video_id),
                {'geo_countries': ['JP']}), 'BrightcoveNew')
```